### PR TITLE
Get alpine package version instead of hash

### PR DESCRIPTION
**Pull request by nemchik**
Use APKINDEX instead of running an alpine container to access apk

### CODE REVIEW

File: roles/generate-jenkins/templates/Jenkinsfile.j2 [hunk 1]
```diff
@@ -142,8 +142,8 @@ pipeline {
         sh '''docker pull alpine:${DIST_TAG}'''
```
Comment (aptalca) [inline roles/generate-jenkins/templates/Jenkinsfile.j2:142]: we can also remove this line

Comment (nemchik) [reply roles/generate-jenkins/templates/Jenkinsfile.j2]: Done

File: roles/generate-jenkins/templates/EXTERNAL_TRIGGER.j2 [hunk 2]
```diff
@@ -33,8 +33,8 @@ jobs:
 {% elif external_type == "github_stable" %}
           EXT_RELEASE=$(curl -u "${{ '{{' }} secrets.CR_USER {{ '}}' }}:${{ '{{' }} secrets.CR_PAT {{ '}}' }}" -sX GET "https://api.github.com/repos/{{ better_vars.EXT_USER }}/{{ better_vars.EXT_REPO }}/releases/latest" | jq -r '. | .tag_name')
 {% elif external_type == "alpine_repo" %}
-          EXT_RELEASE=$(docker run --rm alpine:{{ better_vars.DIST_TAG }} sh -c 'apk update --quiet --repository '"{{ better_vars.DIST_REPO }}"'\
-            && apk info --repository '"{{ better_vars.DIST_REPO }}"' '"{{ better_vars.DIST_REPO_PACKAGES }}"' | md5sum | cut -c1-8')
+          EXT_RELEASE=$(curl -sL "{{ better_vars.DIST_TAG }}x86_64/APKINDEX.tar.gz" | tar -xz \
```
Comment (aptalca) [inline roles/generate-jenkins/templates/EXTERNAL_TRIGGER.j2:36]: This should be `DIST_REPO` instead of  `DIST_TAG`

Comment (nemchik) [reply roles/generate-jenkins/templates/EXTERNAL_TRIGGER.j2]: Done. Good catch.

**Comment by nemchik** [comment]
This is tested and successfully worked as intended with https://github.com/linuxserver/docker-mariadb/pull/66/commits/fa1a7143f284b715d31d648bcf34b86b2a500c77

**Review by aptalca — APPROVED** [review]

**Pull request merged**